Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -141,13 +143,13 @@ def test_select_font_mouse(self):
         fontlist = dialog.fontlist
         fontlist.activate(0)
 
-      # Select next item in listbox
+        # Select next item in listbox
         fontlist.focus_force()
         fontlist.see(1)
         fontlist.update()
         x, y, dx, dy = fontlist.bbox(1)
-        fontlist.event_generate('<Button-1>', x=x+dx//2, y=y+dy//2)
-        fontlist.event_generate('<ButtonRelease-1>', x=x+dx//2, y=y+dy//2)
+        fontlist.event_generate('<Button-1>', x=x + dx // 2, y=y + dy // 2)
+        fontlist.event_generate('<ButtonRelease-1>', x=x + dx // 2, y=y + dy // 2)
```

Comment:
After re-reading PEP8, I concur with your way to present `x = x + dx//2`, but it's weird in this situation, too.

I convert it to another way to represent:
```
x += dx // 2
y += dy // 2
event_generate(x=y, y=y)
```

I hope this will be more general in this case.